libs/features/mas/dist/mas.css

```diff
@@ -0,0 +1,170 @@
+  --color-blue-800: #242424;
+  --color-black: #000;
+  --color-accent: var(--link-color-dark);
+  --color-accent-hover: var(--link-hover-color-dark);
+
+  -webkit-font-smoothing: antialiased;
+}
+
+/* Con Button */
+.con-button {
+  background-color: transparent;
+  border-radius: 16px;
+  border: 2px solid var(--text-color);
+  color: var(--text-color);
+  display: inline-block;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 16px;
+  padding: 5px 14px;
+  text-decoration: none;
+}
+
+.con-button:hover {
+  background-color: var(--color-black);
+  border-color: var(--color-black);
+  color: var(--color-white);
+  text-decoration: none;
+}
+
+.con-button.button-s {
+  border-radius: 12px;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 15.02px;
+  padding: 3px 12px;
+}
+
+.con-button.button-l {
+  border-radius: 20px;
+  font-size: 17px;
+  line-height: 20px;
+  min-height: 21px;
+  padding: 7px 18px 8px;
+}
+
+.con-button.button-xl {
+  border-radius: 25px;
+  font-size: 19px;
+  line-height: 24px;
+  min-height: 28px;
+  padding: 10px 24px 8px;
+}
+
+.con-button.button-justified {
+  display: block;
+  text-align: center;
+  width: 100%;
+}
+
+.dark .con-button {
+  border-color: var(--color-white);
+  color: var(--color-white);
+}
+
+.light .con-button {
+  border-color: var(--text-color);
+  color: var(--text-color);
+}
+
+.light .con-button:hover,
+.light .con-button:active {
+  background-color: var(--color-black);
+  border-color: var(--color-black);
+  color: var(--color-white);
+}
+
+.dark .con-button:hover {
+  background-color: var(--color-white);
+  color: var(--color-black);
+  text-decoration: none;
+}
+
+.con-button.blue {
+  background: var(--color-accent);
+  border-color: var(--color-accent);
+  color: var(--color-white);
+}
+
+.con-button.blue:hover,
+.con-button.blue:active {
+  background: var(--color-accent-hover);
+  border-color: var(--color-accent-hover);
+  color: var(--color-white);
+}
+
+.con-button.fill {
+  background: var(--text-color);
+  border-color: var(--text-color);
+  color: var(--color-white);
+}
+
+.con-button.fill:hover,
+.con-button.fill:active {
+  background: var(--color-black);
+  border-color: var(--color-black);
+}
+
+.dark .con-button.fill {
+  background: var(--color-white);
+  border-color: var(--color-white);
+  color: var(--text-color);
+}
+
+.dark .con-button.fill:hover,
+.dark .con-button.fill:active {
+  background: var(--color-white);
+  border-color: var(--color-white);
+  color: var(--color-black);
+}
+
+/* Disabled State */
+.con-button[aria-disabled="true"],
+.con-button.blue[aria-disabled="true"],
+.con-button.fill[aria-disabled="true"] {
+  background-color: transparent;
+  border-color: var(--color-gray-500);
+  color: var(--color-gray-500);
+}
+
+.con-button.blue[aria-disabled="true"],
+.con-button.fill[aria-disabled="true"] {
+  background-color: var(--color-gray-500);
+  color: var(--color-gray-300);
```

Comment:
Is this any different from https://github.com/adobecom/milo/pull/3306/files#diff-ed23b36063b3814e2ce691dcf650d3aefaf5c9fa9d7a9d8336113140768e261dR146 ? Can we re-arrange styles so the code is not duplicated?